conf/maestro-conf/edge-config-dell5000-demo.yaml

```diff
@@ -63,12 +57,6 @@ var_defs:
      value: "{{SNAP}}/wigwag/etc/versions.json"
    - key: "UPGRADE_VERSIONS_FILE"
      value: "{{SNAP}}/wigwag/etc/versions.json"
-devicedb_conn_config:
```

Comment:
Are these devicedb configuration linked with built-in dhcp client?

Reply:
I don't know. I was just trying to disable the DHCP from clashing with the Dell 5000's built-in. How do I tell?

Reply:
I think commenting just the network portion will disable the built-in dhcp client. Are we going to use built-in client on Dell 5000? If you want to temporary disable it then I recommend we create another configuration file with your changes and use that to build developer snaps.